src/FunctionApps/BulkImport/Import/main.py

```diff
@@ -0,0 +1,98 @@
+import logging
+import os
+import zipfile
+
+import requests
+import multiprocessing
+import json
+from requests.adapters import HTTPAdapter
+from requests.packages.urllib3.util.retry import Retry
+
+unzipped_directory = "./FhirResources"
+
+def main(zip_file):
+    unzip_input_file(zip_file)
+    process_ndjson_files(unzipped_directory)
+
+def unzip_input_file(zip_file):
```

Comment:
I think we'll ultimately want doc strings for each function that give types for both inputs and outputs, as well as provide a summary of the method's invocation. e.g.

```
def unzip_input_file(zip_file: str) -> str:
    '''
    Opens the given file and unzips to a new directory for processing.
    '''
    [code here]
```